src/t_list.c

```diff
@@ -892,7 +902,22 @@ void lmoveCommand(client *c) {
         != C_OK) return;
     if (getListPositionFromObjectOrReply(c,c->argv[4],&whereto)
         != C_OK) return;
-    lmoveGenericCommand(c, wherefrom, whereto);
+    lmmoveGenericCommand(c,c->argv+1,1,c->argv[2],wherefrom,whereto);
+}
+
+/* LMMOVE <destination> <numkeys> <srckey> [<srckey> ...] (LEFT|RIGHT) (LEFT|RIGHT) */
+void lmmoveCommand(client *c) {
+    long numkeys = 0;
+    if (getRangeLongFromObjectOrReply(c,c->argv[2],c->argc-5,c->argc-5,
```

Comment:
Having defined constraints in `getRangeLongFromObjectOrReply` is a bit uncommon for the redis sources. But I don't see any reason to add more separate checks.